src/test/blk_recovery/blk_recovery.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014-2018, Intel Corporation
+ * Copyright 2014-2019, Intel Corporation
```

Comment:
this seems like an unrelated change.

Reply:
I tested this test under memcheck, pmemcheck, drd, helgrind and asan. It always passes, because right now the part that is crashing is executed with expect_abnormal_exit.

(force-disables are not needed, so I removed them)